src/clarity-angular/datagrid/datagrid-action-overflow.ts

```diff
@@ -61,6 +70,11 @@ export class DatagridActionOverflow {
             //Start checking if current and actionMenuHost are equal. If not traverse to the parentNode and check again.
             while (current) {
                 if (current === actionMenuHost) {
+                    // if this datagrid has checkboxes, uncheck all but this current one
+                    if (this.selection.selectable) {
+                        this.selection.current = [];
+                        this.selection.setSelected(this.item, true);
+                    }
```

Comment:
Ok, we need to talk about this. This is really not the behavior I'm expecting from the UX specs.

From what I see, actions in the overflow menu should be completely decoupled from selection, they just allow instant actions on the model of a row. Which also means you shouldn't need the model, since you already have it from the row. 

But even if we have ways to grab the model, I think this behavior is too quirky and too strict. Basically, you're going for this:
```html
<clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
    <clr-dg-action-overflow [clrDgActionItem]="user">
        <a class="action-item" (click)="onEdit()">Edit</a>
    </clr-dg-action-overflow>
    …
</clr-dg-row>
```
but I believe this is more natural, requires the user to learn less inputs/outputs, and is more flexible at the same time:
```html
<clr-dg-row *clrDgItems="let user of users" [clrDgItem]="user">
    <clr-dg-action-overflow>
        <a class="action-item" (click)="onEdit(user)">Edit</a>
    </clr-dg-action-overflow>
    …
</clr-dg-row>
```